### PR TITLE
Fix #6352 remove flask and uwsgi

### PR DESCRIPTION
**Pull request by robnagler**
- Removed db upgrades from 2022 and before

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -7,50 +7,44 @@
...
+        "pyIsEmail",
+        "pykern",
+        "pytest-asyncio",
+        "pytz",
+        "requests",
+        "tornado",
+        "user-agents",
+        # only needed for testing (srunit)
+        "websockets",
+        "werkzeug==2.0.3",
```
Comment (e-carlin) [inline setup.py:33]: Can this be removed too?

File: sirepo/db_upgrade.py [hunk 2]
```diff
@@ -63,143 +66,11 @@ def _20210211_add_flash_proprietary_lib_files(qcall, force=False):
             # Add's the flash proprietary lib files (unpacks flash.tar.gz)
             sirepo.sim_data.audit_proprietary_lib_files(
                 qcall=qcall,
-                force=force,
+                force=True,
                 sim_types=set(("flash",)),
             )
 
 
-def _20210211_upgrade_runner_to_job_db(qcall):
```
Comment (mkeilman) [inline sirepo/db_upgrade.py:71]: Out of curiosity, what governs (or should govern) when db upgrades are obsolete and can be removed?

**Review by mkeilman — APPROVED** [review]
lgtm - and satisfying to ditch 500 lines of code

File: sirepo/db_upgrade.py (hunk 2 above)
Comment (robnagler) [reply sirepo/db_upgrade.py:71]: This was a judgment call. We don't have any installations using the old runner, that's for sure. The spa session probably is the only one that's a "problem", but it is a delete, not an add, so not critical if it doesn't get done. Flash is proprietary and was technically a duplicate (see the code) so seemed like only one upgrade was necessary.

File: setup.py (hunk 1 above)
Comment (robnagler) [reply setup.py]: Yup 9b57b10df

**Pull request merged**